runners/core-construction-java/build.gradle

```diff
@@ -38,10 +38,12 @@ dependencies {
   compile project(path: ":sdks:java:core", configuration: "shadow")
   compile library.java.vendored_grpc_1_21_0
   compile library.java.vendored_guava_26_0_jre
+  compile library.java.classgraph
   compile library.java.jackson_core
   compile library.java.jackson_databind
   compile library.java.joda_time
   compile library.java.slf4j_api
+  testCompile project(path: ":sdks:java:core", configuration: "testRuntime")
```

Comment:
Lets move this to the bottom along with another "testCompile project(...".

Reply:
done